packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -159,14 +169,23 @@ export type FetchErc20AllowanceParams = FetchErc20DataProps & {
  */
 export async function fetchErc20Allowance(params: FetchErc20AllowanceParams) {
   const { address, provider, owner, spender } = params
-
-  // todo: fall back if there is no multicall?
-  const multiCaller = await MultiCaller.fromProvider(provider)
-  const [tokenData] = await multiCaller.getTokenData([address], {
-    allowance: { owner, spender }
-  })
-
-  return tokenData?.allowance ?? constants.Zero
+  try {
+    const multiCaller = await MultiCaller.fromProvider(provider)
+    const [tokenData] = await multiCaller.getTokenData([address], {
+      allowance: { owner, spender }
+    })
+    return tokenData?.allowance ?? constants.Zero
+  } catch (e) {
+    //  fall back if there is no multicall
+    const erc20 = ERC20__factory.connect(address, provider)
+    try {
+      const allowance = await erc20.allowance(owner, spender)
+      return allowance ?? constants.Zero
+    } catch (err) {
+      // contract is not a valid Erc20 token, exit
```

Comment:
Maybe we should have some logs here and return fallback. I'm not sure we want to throw error if it's not handled later

Reply:
Updated to capturing the error, and returning a fallback value.